src/test/java/uk/gov/hmcts/ccd/data/casedetails/search/SearchDirectionTest.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.ccd.data.casedetails.search;
+
+import java.util.Optional;
+
+import org.junit.Test;
+
+import org.junit.Assert;
+
+public class SearchDirectionTest {
```

Comment:
please update to SortDirectionTest

Reply:
My silly semantic dyslexia.. :)
Corrected. Thank you.